package.json

```diff
@@ -25,7 +25,7 @@
     "changelog": "node ./bin/cli.js",
     "clean": "rimraf lib",
     "fix": "npm run lint -- --fix",
-    "lint": "tslint --project . --format stylish",
+    "lint": "eslint 'src/**/*.ts' --format stylish",
```

Comment:
do we not have to set something like `--ext ts`?

```suggestion
    "lint": "eslint src --ext ts --format stylish",
```

Reply:
Difference between the debug outputs:
* `eslint 'src/**/*.ts' --format stylish`: [output-src.txt](https://github.com/lerna/lerna-changelog/files/3833217/output-src.txt)
* `eslint src --ext ts --format stylish`: [output-ext.txt](https://github.com/lerna/lerna-changelog/files/3833239/output-ext.txt)

The out is same, but latter one is readable so going ahead with it
![output-diff](https://user-images.githubusercontent.com/16024985/68620945-70ec9080-0483-11ea-9e5a-d4d491693c7b.png)